libraries/ms-drivers/inc/spv1020_mppt_defs.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+// Internal SPV1020 command definitions and bitmasks.
+// Datasheet: https://media.digikey.com/pdf/Data%20Sheets/ST%20Microelectronics%20PDFS/SPV1020.pdf
+
+#define COMMAND_NOP 0x01
+#define COMMAND_SHUT 0x02
+#define COMMAND_TURN_ON 0x03
+#define COMMAND_READ_CURRENT 0x04
+#define COMMAND_READ_VIN 0x05
+#define COMMAND_READ_PWM 0x06
+#define COMMAND_READ_STATUS 0x07
+
+// Bitmasks for extracting information from the status byte.
+#define OVC_BITMASK 0b1111000
+#define OVV_BITMASK 0b0000100
+#define OVT_BITMASK 0b0000010
+#define CR_BITMASK 0b0000001
```

Comment:
Typically in driver code (and in general) we prefix these. In thise case I guess it'd be `SPV1020_` may or may not want to shorten BITMASK to MASK and COMMAND to CMD to compensate.